build.sbt

```diff
@@ -47,13 +47,12 @@ ThisBuild / libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml"
 
 Global / semanticdbEnabled := !(Global / insideCI).value
 // Change main/src/main/scala/sbt/plugins/SemanticdbPlugin.scala too, if you change this.
-Global / semanticdbVersion := "4.9.9"
-Global / excludeLintKeys += Utils.componentID
-Global / excludeLintKeys += scriptedBufferLog
-Global / excludeLintKeys += checkPluginCross
-Global / excludeLintKeys += nativeImageVersion
-Global / excludeLintKeys += nativeImageJvm
-ThisBuild / evictionErrorLevel := Level.Info
+Global / semanticdbVersion := "4.7.8"
```

Comment:
Is there a particular reason to change `Global / semanticdbVersion` from "4.9.9" to "4.7.8"?

Reply:
I think this just came in with merge automatically. We'd need to force everyone to patch develop first if we want to prevent this type of things.

Reply:
Yeah sometimes git does weird thing.

(On develop)
https://github.com/sbt/sbt/blob/8dcdee8732b90e7612e232719d683d04ef372c05/build.sbt#L50

(On 1.10.x)
https://github.com/sbt/sbt/blob/66935c460d9ab4b9ccaed241807a90a2a4eb7c44/build.sbt#L51

Reply:
Addressed in 12db63951b9a34c196622dbf7b8a7fdfe679435a